src/sidebar/services/api.js

```diff
@@ -226,30 +217,70 @@ export class APIService {
         onRequestFinished: store.apiRequestFinished,
       });
 
+    // Define available API calls.
+    //
+    // The type syntax is APICall<Parameters, Body, Result>, where `void` means
+    // no body / empty response.
+
+    /**
+     * @typedef AnnotationSearchResult
```

Comment:
The naming here and, below, `GroupListParams`, is helpful in tying it back to `Result` and `Params`, respectively. 👍🏻